src/Traits/LucidTrait.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Vinelab\Tracing\Traits;
+
+use Illuminate\Log\Events\MessageLogged;
+use Illuminate\Support\Facades\Event;
+use Lucid\Foundation\Events\FeatureStarted;
```

Comment:
"Where is this namespace coming from?" - is what I would have asked if never heard about lucid before.

You should at least mention [lucid-arch/laravel-foundation](https://packagist.org/packages/lucid-arch/laravel-foundation) package in composer.json under `suggest` option.